### PR TITLE
feat: add prompts to add nx to angular cli

### PR DESCRIPTION
**Pull request by Cammisuli**

### CODE REVIEW

File: libs/vscode/nx-conversion/src/lib/vscode-nx-conversion.spec.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { vscodeNxConversion } from './vscode-nx-conversion';
+
+describe('vscodeNxConversion', () => {
+  it('should work', () => {
+    expect(vscodeNxConversion()).toEqual('vscode-nx-conversion');
+  });
+});
```
Comment (ZackDeRose) [inline libs/vscode/nx-conversion/src/lib/vscode-nx-conversion.spec.ts:7]: Want to remove the spec for now?

File: libs/vscode/nx-conversion/src/lib/vscode-nx-conversion.ts [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+import {
+  commands,
+  ExtensionContext,
+  Task,
+  tasks,
+  TaskScope,
+  window,
+} from 'vscode';
+
+const PROMPT_COUNT = 1;
```
Comment (ZackDeRose) [inline libs/vscode/nx-conversion/src/lib/vscode-nx-conversion.ts:14]: Think we wanna bump this back up to 5 right?

File: libs/vscode/nx-conversion/src/lib/vscode-nx-conversion.ts [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+const PROMPT_MSG = 'Convert to Nx?';
```
Comment (ZackDeRose) [inline libs/vscode/nx-conversion/src/lib/vscode-nx-conversion.ts:15]: I was thinking for more along the lines of "Would you like to upgrade the Angular CLI with Nx (making your `ng` commands up to 10 times faster)?"

File: package.json [hunk 4]
```diff
@@ -26,7 +26,8 @@
     "@monodon/typescript-nx-imports-plugin": "0.2.0",
     "@yarnpkg/fslib": "2.6.1-rc.5",
     "@yarnpkg/libzip": "2.2.3-rc.5",
-    "jsonc-parser": "^3.0.0"
+    "jsonc-parser": "^3.0.0",
+    "tslib": "^2.0.0"
```
Comment (ZackDeRose) [inline package.json:30]: was this intended to be added?

**Review by ZackDeRose — APPROVED** [review]
Couple small questions - other than that lgtm

File: libs/vscode/nx-conversion/src/lib/vscode-nx-conversion.ts (hunk 3 above)
Comment (Cammisuli) [reply libs/vscode/nx-conversion/src/lib/vscode-nx-conversion.ts:15]: Yup, we'll need to think of a good tag line

File: libs/vscode/nx-conversion/src/lib/vscode-nx-conversion.ts (hunk 2 above)
Comment (ZackDeRose) [inline libs/vscode/nx-conversion/src/lib/vscode-nx-conversion.ts]:
```suggestion
const PROMPT_COUNT = 5;
```

File: apps/vscode/src/main.ts [hunk 5]
```diff
@@ -53,6 +52,7 @@ import {
   ProjectJsonSchema,
 } from '@nx-console/vscode/json-schema';
 import { enableTypeScriptPlugin } from '@nx-console/typescript-plugin';
+import { NxConversion } from '@nx-console/vscode/nx-conversion';
```
Comment (ZackDeRose) [inline apps/vscode/src/main.ts:55]: Looks like lint error here

Comment (ZackDeRose) [reply apps/vscode/src/main.ts:55]: Looks like just an error in my IDE

**Pull request merged**